source/index.md

```diff
@@ -31,7 +31,7 @@ Specifications for SMIRNOFF and other standards defined by OpenFF can be found a
 : Submit and retrieve datasets with rich metadata from QCFractal instances
 
 [`openff-fragmenter`](https://docs.openforcefield.org/projects/fragmenter)
-: Fragment molecules for efficient quantum mechanical torsion scans 
+: Fragment molecules for efficient quantum mechanical torsion scans
```

Comment:
https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#trailing-whitespace would standardize against what our different IDEs (and developers) sneak or accidentally let in